src/pull.js

```diff
@@ -0,0 +1,94 @@
+/* @flow */
+
+import printit from 'printit'
+
+import RemoteCozy from './remote/cozy'
+import Prep from './prep'
+import Pouch from './pouch'
+import * as conversion from './conversion'
+
+import type { RemoteDoc } from './remote/document'
+import type { LocalDoc } from './local/document'
+
+const log = printit({
+  prefix: 'Pull',
+  date: true
+})
+
+// Pull changed files and directories metadata from the remote Cozy, and merge
+// them into the local database.
+export default class Pull {
```

Comment:
I'm not sure it's a good idea to call this `Pull`. There is a pull command on the command-line that does a bit more:

- it track changes on the remote cozy and update the pouchdb with these changes (like your class)
- it also applies these changes to the file-system